src/components/AssetsTable/AssetsTable.vue

```diff
@@ -0,0 +1,317 @@
+<template>
+  <table class="assets-table">
+    <thead class="assets-table__header">
+      <tr>
+        <th
+          v-for="(column, index) of columns"
+          :key="index"
+          class="assets-table__header-item"
+          @click="sortTable(column.label)"
+          :class="{
+            sortable: column.sortable !== false,
+            asc: this.setSortType('asc', column),
+            desc: this.setSortType('desc', column),
+            'sort-active': sortConfig.column === column.label,
```

Comment:
this is ok, but just FYI: https://vuejs.org/style-guide/rules-recommended.html#element-attribute-order
These recommendations will make your components a bit more readable